salesforce-gui/src/main/java/org/fundacionjala/salesforce/ui/pageObjects/account/accountImportPage/bulkDataLoadJobs/BulkDataLoadJobsPage.java

```diff
@@ -0,0 +1,98 @@
+package org.fundacionjala.salesforce.ui.pageObjects.account.accountImportPage.bulkDataLoadJobs;
+
+import org.fundacionjala.core.selenium.interaction.GuiInteractioner;
+import org.fundacionjala.salesforce.ui.pageObjects.commonPages.BasePage;
+import org.fundacionjala.salesforce.utils.CSVUtil;
+import org.fundacionjala.salesforce.utils.FileUtils;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class BulkDataLoadJobsPage extends BasePage {
+
+    private static final By HEADER_TABLE = By.xpath("//table[@class='detailList']/*/tr/th");
+    private static final By CONTEND_TABLE = By.xpath("//table[@class='detailList']/*/tr/td/span");
+    private static final String DOWNLOAD_OPTION = "//a[text()='%s']";
+    private static final String DOWNLOAD_PATH = System.getProperty("user.dir") + "/src/test/resources/tmp/";
+    private List<String> fileList;
+    private static final int WAIT_TIME = 3000;
+    private static final int NUMBER_THREE = 3;
+
+
+    /**
+     * Gets all data of the table Bulk Data Load Job.
+     *
+     * @return a map with the all data of the table
+     */
+    public Map<String, String> bulkDataLoadJobDetail() {
+        Map<String, String> map = new HashMap<>();
+        List<WebElement> headerList = getDriver().findElements(HEADER_TABLE);
+        List<WebElement> contendList = getDriver().findElements(CONTEND_TABLE);
+
+        for (int i = 0; i < headerList.size(); i++) {
+            String key = headerList.get(i).getText();
+            String value = contendList.get(i).getText();
+            map.put(key, value);
+        }
```

Comment:
You can reduce this to only one line:   ----->     map.put(headerList.get(i).getText(), contendList.get(i).getText())